docs/why-cadence.md

```diff
@@ -0,0 +1,16 @@
+## Why should you use Cadence for building smart contracts ?
```

Comment:
```suggestion
---
title: Why Use Cadence?
---

## Why should you use Cadence for building smart contracts ?
```

Reply:
I'd prefer to use the more descriptive title, the reasons are revolving around the benefits specifically for smart-contract development.